src/node.js

```diff
@@ -46,7 +46,9 @@ const pluginFactory: () => PluginType = () =>
         translate(key, interpolations = {}) {
           const template = this.translations[key];
           return template != null
-            ? template.replace(/\${(.*?)}/g, (_, k) => interpolations[k])
+            ? template.replace(/\${(.*?)}/g, (_, k) =>
+                interpolations[k] ? interpolations[k] : '${' + k + '}'
```

Comment:
I think an explicit `interpolations[k] === void 0` check here would be better.

- I think replacing with an empty string (which would coerce to `false`) is potentially a valid use case
- While input should probably always be a string, passing in the number `0` seems not totally unreasonable (which would coerce to `false`)

Reply:
Updated.